bindings/rascal/models/krr.py

```diff
@@ -81,7 +81,7 @@ def predict(self, managers, compute_gradients=False):
             return Y0 + np.dot(KNM, self.weights).reshape((-1))
         else:
             rep = self.kernel._representation
-            gradients = -compute_forces(rep, self.kernel._kernel, managers.managers, self.X_train._sparse_points, self.weights.reshape((1, -1)))
+            gradients = -compute_sparse_kernel_forces(rep, self.kernel._kernel, managers.managers, self.X_train._sparse_points, self.weights.reshape((1, -1)))
```

Comment:
why is there a minus here? Shouldn't the C++ code produce results directly with the right sign?

Reply:
`compute_sparse_kernel_forces` gives forces while we want gradients at the model level in python to keep the logic of the model.

Reply:
Should their be a `compute_sparse_kernel_gradient` instead then?

Reply:
I don't think it is needed

Reply:
I think the point is that it seems a bit odd to flip the sign of these quantities _twice_ -- once (implicitly) when computing forces, since those are the negative gradients of the atomic energies, and once here to get _back_ to gradients again (and a third time when whatever MD program we're using requests forces, not gradients!). So I think we should be more consistent with either sticking with forces or with gradients, so that we don't get confused by too many sign flips. My vote is to stick with forces.